File: Java/070. Meses/Meses.java
import java.util.Scanner;
public class Meses
{
    public static void main(String[] args)
    {
        Scanner scan = new Scanner(System.in);
        int mes = scan.nextInt();
        String[] meses = {"JANEIRO", "FEVEREIRO", "MARCO", "ABRIL", "MAIO", 
                         "JUNHO", "JULHO", "AGOSTO", "SETEMBRO", 
                         "OUTUBRO", "NOVEMBRO", "DEZEMBRO"}; 
        if (mes < 1 || mes > 12) {
            System.out.println("MES INVALIDO");
        } else {
            System.out.println(meses[mes - 1]);
        }
        scan.close();
    }
}

File: C++/070. Meses/Meses.cpp
#include <iostream>
int main()
{
    int M;
    std::cin >> M;
    std::string meses[] = 
    {
        "JANEIRO", "FEVEREIRO", "MARCO", "ABRIL", "MAIO", "JUNHO",
        "JULHO", "AGOSTO", "SETEMBRO", "OUTUBRO", "NOVEMBRO", "DEZEMBRO"
    };
    if (M >= 1 && M <= 12) {
        std::cout << meses[M - 1] << std::endl;
    } else {
        std::cout << "MES INVALIDO" << std::endl;
    }
}